### ISSUE TITLE
Add "Deploy to Heroku" button

### ISSUE THREAD

**Issue by laksh22** [scorelab/LabelLab#256]
**Is your feature request related to a problem? Please describe.**
Setting up Labellab for users might be too cumbersome if they have to set it up manually. Adding a one-click deploy button to the README will enable anyone to run Labellab for free.

**Describe the solution you'd like**
A one-click __Deploy to Heroku__ button should be added to the README

**Describe alternatives you've considered**
N/A

**Additional context**
I am currently working on this feature.

**Comment by carpecodeum** [comment]
hey @laksh22 i think this feature should be implemented once the machine-learning part of the app  is complete. although its a good feature to add.

**Comment by laksh22** [comment]
Hi @carpecodeum, thanks for the feedback! Even I think that it'll be useful for end users when the machine learning part is implemented. I still think that it can benefit some people who are thinking of contributing and want to test out the app before setting it up locally.

**Comment by laksh22** [comment]
@Tushar19varshney @UdeshUK I am able to deploy LabelLab on Heroku but the only problem now is that file uploads don't work on Heroku so users won't be able to upload images. I was thinking that the free tier of Cloudinary can be added to the Heroku app. Should I go ahead with this?

**Comment by UdeshUK** [comment]
@laksh22 Initial plan was to deploy the server on a GCP Compute Engine. But since there is a maintenance cost it would be ok to deploy on Heroku free tier and Cloudinary for images. 

P.S. But these circumstances can be changed in the future.

**Comment by laksh22** [comment]
Thanks @UdeshUK. For adding the Cloudinary support, should I make a separate issue and PR or just integrate it into the current one?

**Comment by Tushar19varshney** [comment]
@laksh22 Great Idea but I think implementing this feature is too soon till we are in staging or production mode.

**Comment by laksh22** [comment]
@Tushar19varshney Sure, I'll keep working on Heroku integration as more features are added to the project.
